core/src/main/java/org/apache/calcite/runtime/Matcher.java

```diff
@@ -63,7 +58,6 @@ private Matcher(Automaton automaton,
         ImmutableBitSet.builder();
     startSetBuilder.set(automaton.startState.id);
     automaton.epsilonSuccessors(automaton.startState.id, startSetBuilder);
-    startSet = startSetBuilder.build();
```

Comment:
It looks like that 'startSet' is used in  'startSetBuilder.build()'.

Reply:
Still, we don't do anything meaningful with `startSet`. We could change the line to `startSetBuilder.build();` if we don't want to change behavior. WDYT?

https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/util/ImmutableBitSet.java#L958-L968

Reply:
`Matcher` is a work in progress, because we only have a basic implementation of `MATCH_RECOGNIZE` right now. I don't know what we will eventually need. We might need `startSetBuilder` when we implement the `classifier()` operator in `MATCH_RECOGNIZE`.

Let's not try to tidy up that code right now. It's a construction site. There are things in it that are not used now that I added for a reason because I was thinking about the whole problem.